File: csharp/bypstest-ser/src-ser/byps/test/api/inherit/BioFruitService.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using byps;

namespace byps.test.api.inherit
{
	
	// checkpoint byps.gen.cs.GenRemoteClass:93
	public interface BioFruitService : BRemote, 
		FruitService {
	
		bool Certify(String param);
		void Certify(String param, BAsyncResult<bool> asyncResult) ;
		Task<bool> CertifyAsync(String param);
		
		
	}
}  // end namespace

File: java/bypstest-api/src/byps/test/api/inherit/BioFruitService.java
package byps.test.api.inherit;

import byps.RemoteException;

/**
 * @BSessionParamType com.wilutions.byps.test.api.auth.SessionInfo
 */
public interface BioFruitService extends FruitService {

  boolean certify(String param) throws RemoteException;
}
